src/InvocationModel.ts

```diff
@@ -120,9 +135,13 @@ export class InvocationModel implements coreTypes.InvocationModel {
         return response;
     }
 
-    async #convertOutput(binding: RpcBindingInfo, value: unknown): Promise<RpcTypedData | null | undefined> {
+    async #convertOutput(
+        invocId: string,
```

Comment:
Nit: We use `invocationId` pretty regularly elsewhere in the code. I think it's better to use the full name to remain consistent.